php_ast.h

```diff
@@ -53,6 +53,7 @@ extern ast_str_globals str_globals;
 #endif
 
 #if PHP_VERSION_ID < 80000
+# define ZEND_ACC_NO_DYNAMIC_PROPERTIES (1 << 13)
```

Comment:
I think we'll want to define this as 0 to avoid unsetting some other flag (these tend to change a lot between versions).

Reply:
Fixed, I just forgot to delete this. We're no longer referencing this constant before php 8.2